bigframes/session/clients.py

```diff
@@ -121,12 +123,27 @@ def bqclient(self):
             bq_info = google.api_core.client_info.ClientInfo(
                 user_agent=self._application_name
             )
+            default_query_job_config = None
+            default_load_job_config = None
+            if self._bq_kms_key_name:
+                default_query_job_config = bigquery.QueryJobConfig(
```

Comment:
Looks like this breaks the BQML flow, in which the kms_key_name needs to be set differently:

```
Cannot set kms key name in jobs with ML DDL statements. For CREATE TABLE statement, use OPTIONS(kms_key_name=...) instead
```

https://pantheon.corp.google.com/bigquery?project=bigframes-dev&j=bq:US:8e13b27c-b938-4d44-95ba-d0e2fcadf218&page=queryresults&ws=!1m5!1m4!1m3!1sbigframes-dev!2s8e13b27c-b938-4d44-95ba-d0e2fcadf218!3sUS

Reply:
Separated the bqml client with the main bq client.

Reply:
FYI it doesn't help if we also set it in DDL OPTIONS, the API simply doesn't like kms_key_name being set in the job_config for DDL statements https://pantheon.corp.google.com/bigquery?project=bigframes-dev&j=bq:US:9dc79963-ff71-4d66-bdf4-ab336c789e80

Reply:
Thank you for verifying.